safe-global/safe-wallet-web
# Feat: Targeted survey banner popup [SW-269]

## What it solves

Resolves: https://www.notion.so/safe-global/Targeted-messaging-popup-banner-1198180fe57380e2a03cd7e4a1317f1e

## How this PR fixes it
- Shows the popup banner when connected as signer of a safe (for now it is for any safe). The CTA leads to Typeform.

## How to test it
- In a fresh private window, open any safe you are the signer of and accept cookies.
- The banner should be shown.
- On clicking the close icon, the banner should never reappear.
- On clicking the "Ask me later" button, the banner should only reappear if a new browser session is started or if 24 hours have passed.
- Clicking the CTA should lead to Typeform

## Screenshots
![image](https://github.com/user-attachments/assets/9e2065e4-fb65-4e89-a8ba-3e751c46a7a4)

![image](https://github.com/user-attachments/assets/7c7eaddd-5b54-46ed-b03b-30d039f1ade2)


## Checklist
